src/js/Node.js

```diff
@@ -9,6 +9,44 @@ define(['./appendNodeFactory', './util'], function (appendNodeFactory, util) {
       return obj === Object(obj);
   }
 
+  function _pathIsInReadOnlyPaths(aimaraPath, readOnlyAimaraPaths) {
+    var currentPath, areEquals;
+
+    for (var iPaths = 0, lPaths=readOnlyAimaraPaths.length; iPaths < lPaths; iPaths++) {
+      currentPath = readOnlyAimaraPaths[iPaths];
+      areEquals = false;
+
+      if (aimaraPath.length == currentPath.length) {
+        areEquals = true;
+        for (var iCurrentPath = 0, lCurrentPath=currentPath.length; iCurrentPath < lCurrentPath; iCurrentPath++) {
+          if (aimaraPath[iCurrentPath] != currentPath[iCurrentPath]) { 
+            areEquals = false;   
+            iCurrentPath = lCurrentPath;
+          }           
+        }       
+      }
+
+      if (areEquals) {
+        return true;
+      }
```

Comment:
I desprefer writing this complicated loop just to compare 2 string arrays. Don't we have here access to underscore or sugar so we can use higher level functions? If we don't I'd rather use Array.join (knowing that these ara string arrays, and that we can find a separator that is not present on this path)